hybrid/HybridSampleApps/NoteSync/build.gradle

```diff
@@ -5,8 +5,9 @@ dependencies {
 }
 
 android {
-  compileSdkVersion 21
-  buildToolsVersion "21.1.1"
+  compileSdkVersion 23
+  buildToolsVersion '23.0.1'
+  useLibrary  'org.apache.http.legacy'
```

Comment:
Is this required here? Won't it be automatically included if it's in the ```build.gradle``` for ```SalesforceSDK``` (chained dependency)?